### ISSUE TITLE
Resolve Welcome Page Accessibility rule violations

### ISSUE THREAD

**Issue by dpaquette** [HTBox/TwoWeeksReady#111]
Sorry about the formatting here. Re-run the playwright tests to get the output locally.

│ (index) │id│impact │description│nodes│
│    0    │     'button-name'      │ 'critical' │                                  'Ensures buttons have discernible text'             │   1   │
│    1    │    'color-contrast'    │ 'serious'  │ 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds' │   1   │
│    2    │      'image-alt'       │ 'critical' │              'Ensures <img> elements have alternate text or a role of none or presentation'               │   1   │
│    3    │ 'page-has-heading-one' │ 'moderate' │            'Ensure that the page, or at least one of its frames contains a level-one heading'             │   1   │
│    4    │        'region'        │ 'moderate' │ 'Ensures all page content is contained by landmarks'   │   1   │
